grpc-gateway/pb/cloudConfiguration.proto

```diff
@@ -11,12 +11,14 @@ message CloudConfigurationResponse {
     // provides a current time of server in nanoseconds.
     int64 current_time = 1;
     string device_onboarding_code_url  = 2;
-    string jwt_claim_owner_id = 3;
+    string jwt_owner_claim = 3;
+    string jwt_device_id_claim = 11;
```

Comment:
You can still renumber, we removed some stuff and v2 was not yet released.

Reply:
fixed